### PR TITLE
Add Romain Tartière in Caen

### PR DESCRIPTION
**Pull request by smortex**
Now that #204 is merged, let's add myself to Caen :wink:

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -15118,6 +15118,89 @@ var data = {
...
       }
+    },
+    {
+      "since": "2017-05-25",
+      "name": "Romain Tartière",
+      "bio": "FreeBSD / Hacker / Sysadmin / FLOSS / BOFH / Vie privée",
+      "picture": "https://www.gravatar.com/avatar/3e8420016e2c1f4c6e7d8f526f96baeb?s=200",
+      "websites": [
+        {
+          "name": "Site personnel",
```
Comment (dadoonet) [inline baggers.js]: Not allowed. See the list of supported names in the README.

File: baggers.js [hunk 2]
```diff
@@ -15118,6 +15118,89 @@ var data = {
...
+          "url": "https://romain.blogreen.org/"
+        },
+        {
+          "name": "Mastodon",
```
Comment (dadoonet) [inline baggers.js]: Not supported

File: baggers.js [hunk 3]
```diff
@@ -15118,6 +15118,89 @@ var data = {
...
+          "url": "https://mamot.fr/@smortex"
+        },
+        {
+          "name": "GitHub",
+          "url": "https://github.com/smortex"
+        },
+        {
+          "name": "Open Hub",
```
Comment (dadoonet) [inline baggers.js]: Not supported

**Review by dadoonet — CHANGES_REQUESTED** [review]
Some few things to correct.

BTW you are using `…` instead of `...` (3 dots). Is that what you want?

**Comment by smortex** [comment]
D'oh, forgot about it while completing… Switched to generic *Web* website name.

**Comment by smortex** [comment]
> BTW you are using … instead of ... (3 dots). Is that what you want?

Yes, indeed… UTF-8 power :wink:

**Review by dadoonet — APPROVED** [review]

**Pull request merged**